content/runner/vm/installation.md

```diff
@@ -41,7 +40,7 @@ The AWS runner is configured using environment variables. This article reference
 
 ## Amazon EC2 prerequisites
```

Comment:
we should have another section even mentioning providers, and where to get more information